NewForce-at-Mountwest/bangazon-site-the-fiery-goats
# Created controllers and checked seed data

# Description

Created all the controllers needed as well as checked and executed seed data.

Fixes # (issue)
Initial project setup.

## Type of change
Created controllers and verified seed data.

- [ x] New feature (non-breaking change which adds functionality)

# Testing Instructions

1. Use git fetch --all
2. Use git checkout initial-setup
3. Change directory into the one that contains BangazonSite.sln
4. Type start BangazonSite.sln to open the project in Visual Studio.
5. Verify that all the controllers are present and contain no errors.
6. Verify that ApplicationDbContext.cs contains the proper seed data.



# Checklist:

- [x ] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [x ] I have commented my code, particularly in hard-to-understand areas
- [x ] My changes generate no new warnings or errors
- [ x] I have added test instructions that prove my fix is effective or that my feature works
